src/views/Feed/FeedExtra.js

```diff
@@ -45,7 +45,6 @@ FeedExtra.propTypes = {
     PropTypes.oneOfType([
       PropTypes.bool,
       PropTypes.arrayOf(PropTypes.node),
-      PropTypes.arrayOf(PropTypes.string),
```

Comment:
Node type covers string type since strings are renderable.  Node is anything that can be rendered.